frontend/src/component/project/Project/ProjectOverview.tsx

```diff
@@ -39,6 +39,17 @@ const ProjectOverview: FC<{
     storageKey?: string;
 }> = ({ storageKey = 'project-overview-v2' }) => {
     const projectOverviewRefactor = useUiFlag('projectOverviewRefactor');
+
+    if (projectOverviewRefactor) {
```

Comment:
decided to split old and new overview with copy pasted code. eventually we'll remove the old one. My main motivation is to get rid of the project overview call in the new code